app/src/main/java/com/duckduckgo/app/browser/applinks/ExternalAppIntentFlagsFeature.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.applinks
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.Toggle
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "externalAppIntentFlags",
+)
+
+// A safeguard for this fix: https://app.asana.com/0/0/1207374732742336/1207383486029585/f
+interface ExternalAppIntentFlagsFeature {
+
+    @Toggle.DefaultValue(false)
```

Comment:
I think this should be a kill-switch, ie. enabled by default and then disable remotely if needed

```suggestion
    @Toggle.DefaultValue(true)
```